src/main/java/io/github/moderngamingworlds_mods/woodenutilities/common/recipes/WoodcutterRecipeBuilder.java

```diff
@@ -21,22 +22,38 @@
     public List<String> requiredMods = new ArrayList<>();
     public List<String> ingredients = new ArrayList<>();
     public String result;
-    public int count;
+    public int count = 1;
 
     public WoodcutterRecipeBuilder(ResourceLocation loc) {
```

Comment:
Make this private since we only want to use it internal with the `create` method.